tests/models/test_detection.py

```diff
@@ -18,7 +18,7 @@ def test_fasterrcnn():
 
 
 def test_fasterrcnn_train(tmpdir):
-    model = FasterRCNN()
+    model = FasterRCNN(pretrained=False)
```

Comment:
I changed all pretrained to False. As CI coverage is not affected by these. It also speeds up by avoiding us to download the weights.